index.html

```diff
@@ -1,168 +1,213 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-	<meta charset="UTF-8">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<link rel="stylesheet" href="style.css">
-	<title>MY PORTFOLIO</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>MY PORTFOLIO</title>
 </head>
+
 <body>
-	<section class="hero-section">
-		<nav class="nav-bar">
-				<a href="#" class="logo">
+    <section class="hero-section">
+        <nav class="nav-bar">
+            <a href="#" class="logo">
 					My Logo
 				</a>
-			<img src="./assets/images/Union.svg" alt="Menu" class="mobile-menu">
-		</nav>
-		<main class="headline">
-			<h1 class="greeting">
-				I’m Valentine Glad to see you!
-			</h1>
-			<p class="personal-details">
-				I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-			</p>
-			<h3 class="connect">
-				LET’S CONNECT
-			</h3>
-			<ul class="social-links">
-				<li><a href="https://twitter.com/blaze_valentine"><img src="./assets/images/twitter_icon.svg" alt="Twitter"></a></li>
-				<li><a href="https://www.linkedin.com/in/valentine-blaze"><img src="./assets/images/Linkedin_icon.svg" alt="Linkedin"></a></li>
-				<li><a href="http://medium.com"><img src="./assets/images/medium_icon.svg" alt="Medium"></a></li>
-				<li><a href="https://github.com/V-Blaze"><img src="./assets/images/github_icon.svg" alt="Github"a></li>
-				<li><a href="http://"><img src="./assets/images/Vector.svg" alt="Vector"></a></li>
-			</ul>
-		</main>
-	</section>
-	<section class="works-section">
-		<div class="works-card">
-			<div class="work-image">
-				<img src="./assets/images/Snapshoot-Portfolio-1.svg" alt="Project-1">
-			</div>
-			<h1 class="project-title">Tonic</h1>
-			<div class="project-details">
-				<h3 class="project-client">CANOPY</h3>
-				<ul>
-					<li class="project-role">Back End Dev</li>
-					<li class="project-year">2015</li>
-				</ul>
-			</div>
-			<p class="project-decription">
-				A daily selection of privately personalized reads; no accounts or sign-ups required.
-			</p>
-			<ul class="project-languages">
-				<li>Html</li>
-				<li>Css</li>
-				<li>Javascript</li>
-			</ul>
-			<button class="project-button">
+            <img src="./assets/images/Union.svg" alt="Menu" class="mobile-menu">
+        </nav>
+        <main class="headline">
+            <h1 class="greeting">
+                I’m Valentine Glad to see you!
+            </h1>
```

Comment:
- [ ] Kindly replace `h2` tag with `h1` here because an html page can only have one `h1` tag and you already have it on the home :thinking: and it is important to follow the [HTML semantic](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) tags due to SEO purpose :+1:
- [ ] It would be nice if your heading should be in center to follow the Figma template to make your project look professional. :+1: You can use text-align: center; to fix this :blush: